lib/google_mail.rb

```diff
@@ -0,0 +1,65 @@
+require "google/apis/gmail_v1"
+require "googleauth"
+require "json"
+require_relative "env_token_store"
+
+class GoogleMail
+  OOB_URI = "urn:ietf:wg:oauth:2.0:oob".freeze
+  APPLICATION_NAME = "GovWifi Smoke Tests".freeze
+
+  SCOPE = [
+    Google::Apis::GmailV1::AUTH_GMAIL_SEND,
+    Google::Apis::GmailV1::AUTH_GMAIL_MODIFY,
+  ].freeze
+
+  def initialize
+    @service = Google::Apis::GmailV1::GmailService.new
+    @service.client_options.application_name = APPLICATION_NAME
+    @service.authorization = authorize
+  end
+
+  def authorize
+    client_id = Google::Auth::ClientId.from_hash JSON.parse(ENV["GOOGLE_API_CREDENTIALS"])
+    token_store = EnvTokenStore.new ENV
+    authorizer = Google::Auth::UserAuthorizer.new client_id, SCOPE, token_store
+    user_id = "default"
+    credentials = authorizer.get_credentials user_id
+    if credentials.nil?
```

Comment:
```return credentials if credentials.present?``` would avoid this block

Reply:
`present?` is Rails only, but can do `unless credentials.nil?`